src/js/plugins/tts/plugin.tts.js

```diff
@@ -92,8 +116,36 @@ BookReader.prototype.buildMobileDrawerElement = (function (super_) {
 BookReader.prototype.initNavbar = (function (super_) {
     return function () {
         var $el = super_.call(this);
-        var readIcon = '';
         if (this.options.enableTtsPlugin && this.ttsEngine) {
+            this.refs.$BRReadAloudToolbar = $(`
+                <div class="BRReadAloudToolbar" style="display:none">
+                    <div class="BRReadAloudToolbar--controls">
+                        <div class="BRToolbarButton playback-rate-container" title="Change Playback Speed">
+                            <img class="icon" src="${this.imagesBaseURL}icon_playback-rate.svg" alt="Playback Speed"/>
+                            <select name="BRReadAloud-rate">
+                                <option value="0.25">0.25x</option>
+                                <option value="0.5">0.5x</option>
+                                <option value="0.75">0.75x</option>
+                                <option value="1.0" selected>1.0x</option>
+                                <option value="1.25">1.25x</option>
+                                <option value="1.5">1.5x</option>
+                                <option value="1.75">1.75x</option>
+                                <option value="2">2x</option>
+                            </select>
+                        </div>
+                        <button class="BRToolbarButton playPause" title="Toggle Pause">
+                            <img class="icon play" src="${this.imagesBaseURL}icon_play.svg" alt="Play" style="display:none">
```

Comment:
suggestion: add inline css to class

Reply:
Putting this specific style in the CSS feels like unnecessary decoupling since this is entirely controlled by the JS.